apis/goharbor.io/v1beta1/harborcluster_types.go

```diff
@@ -237,6 +238,8 @@ type StorageSpec struct {
 	Azure *AzureSpec `json:"azure,omitempty"`
 	// +kubebuilder:validation:Optional
 	Gcs *GcsSpec `json:"gcs,omitempty"`
+	// +kubebuilder:validation:Optional
+	OSS *OssSpec `json:"oss,omitempty"`
```

Comment:
```suggestion
	Oss *OssSpec `json:"oss,omitempty"`
```

Reply:
why this is desired? I feel this is similar as `Gcs` which is also a abbreviation.